docs/security/host-certs.md

```diff
@@ -61,30 +70,33 @@ The INI file contains the following information:
 -   Host URL
 
 Validate authentication
-============================
+------------------------
 
-Make sure you can create a valid grid proxy. To do so, please follow instructions:
+Make sure you can create a valid grid proxy. To do so, please follow
+instructions:
 
 1.  Double-check your configuration to make sure that you are allowed (specifically, your credentials are allowed) to access your CE. You can be either member of a VO allowed to run at the site or you can add your personal certificate locally. This will either be in your GUMS server or in your `edg-mkgridmap` configuration.
-2.  Create a proxy with `voms-proxy-init` or `grid-proxy-init`. For example:
+2.  Create a proxy with `voms-proxy-init` or `grid-proxy-init`. For example (substitute your own VO) :
```

Comment:
Is there any reason why a VOMS proxy is needed?  Seems like it's just something that can fail.  How about doing only `voms-proxy-init` with no VO?

Reply:
Yeah, a voms-proxy-init without a VO should work to validate the user cert.  Changed the invocation.  There is a FAQ that essentially repeats this,  should I drop that?

Reply:
In that case, you'll also want to remove ` (substitute your own VO) `